oxts_ins/include/oxts_ins/tf2_geometry_msgs.hpp

```diff
@@ -0,0 +1,1231 @@
+// Copyright 2008 Willow Garage, Inc.
```

Comment:
Will this mess up other packages that relies on any local installations of tf2_geometry_msgs? In the dataspeedDockerEnv repo, some packages use tf2_geometry_msgs, not just this package which is a submodule there.

Reply:
I don't think so. If you use this package there this file is already added like this.

Reply:
Then I'm happy